client/components/Help/Help.js

```diff
@@ -1,5 +1,6 @@
 import React, { Component, PropTypes} from 'react';
```

Comment:
what the hell is going on with spaces in import statements?? fix please
and add rule to eslint.rc
    'object-curly-spacing': ['error', 'always'],